pset-iterators.js

```diff
@@ -65,6 +69,16 @@ Expected Result Shape: An array of objects (with each object representing a pres
 console.log('Problem 2:')
 
 // Add your code below this line
+//use filter to focus on each president's names
+const presidentsNamedJames = presidents.filter((president) => {
+  //use split to separate the names, use index of 0 to access the first name
+  //use === to check if the first name is equal to james
+  return president.president.split(' ')[0] === 'James'
```

Comment:
Great use of the filter method; I like how you ran your logic and returned based on the conditional all in one line.